src/components/EventHighlight/EventHighlight.js

```diff
@@ -93,6 +103,16 @@ function EventHighLight({ classes }) {
 				<Typography variant="body1">
 					{event.description}
 				</Typography>
+				<Link href={event.link}
+					target="_blank" rel="noopener noreferrer"
+					underline="none">
+					<Button variant="outlined"
+						classes={{ root: classes.button }}
+					>
+						{event.button}
+						<LaunchIcon classes={{ root: classes.icon }}/>
```

Comment:
```suggestion
						<LaunchIcon classes={{ root: classes.icon }} />
```

We always keep a space before `/` for a self-enclosing tag.